data_summaries/PII-masking-200k.json

```diff
@@ -0,0 +1,55 @@
+{
+    "pii-masking-200k": {
+        "Unique Dataset Identifier": "pii-masking-200k",
+        "Dataset Name": "pii-masking-200k",
+        "Paper Title": "",
+        "Dataset URL": "https://huggingface.co/datasets/ai4privacy/pii-masking-200k",
+        "GitHub URL": "",
+        "Hugging Face URL": "https://huggingface.co/datasets/ai4privacy/pii-masking-200k",
+        "Papers with Code URL": "",
+        "ArXiv URL": "",
+        "Semantic Scholar Corpus ID": "",
+        "Collection": "PII-Masking-200k",
+        "Collection URL": "https://huggingface.co/ai4privacy",
+        "Languages": [
+            "English",
+            "French",
+            "German",
+            "Italian"
+        ],
+        "Task Categories": [
+            "Token Classification",
+            "Text Generation"
+        ],
+        "Text Sources": [
+            "business letters",
+            "education",
+            "psychology",
+            "huggingface.co/datasets/ai4privacy/pii-masking-43k",
+            "huggingface.co/datasets/ai4privacy/pii-masking-65k"
+        ],
+        "Model Generated": [],
+        "Format": [
+            "Chain-of-Thought"
+        ],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [
+            "huggingface.co/datasets/ai4privacy/pii-masking-43k",
+            "huggingface.co/datasets/ai4privacy/pii-masking-65k"
+        ],
+        "Creators": [
+            "AI4Privacy"
+        ],
+        "Licenses": [
+            {
+                "License": "Various",
+                "License URL": "https://huggingface.co/datasets/ai4privacy/pii-masking-200k/blob/main/license.md"
```

Comment:
Robert Mahari (rmahari) will add this as a custom license and create the labels, since it should allow researcher access/non-commercial use, from my read. Could you reach out to him on discord and ask?

Reply:
This is a new one! They have a two-tier commercial license with no requirements for small companies (<3 employees). For the same of simplicity, I'd say this is a custom license that's non-commercial (NC), no share alike, and no attribution.

Reply:
Thank you @rmahari. So in this case, the license will be NC or custom license of NC